lib/Migration/Version1100Date20210406144257.php

```diff
@@ -21,81 +21,13 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		/** @var ISchemaWrapper $schema */
 		$schema = $schemaClosure();
 
-		$table = $schema->getTable('mail_accounts');
-		$table->changeColumn('provisioned', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('show_subscribed_only', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('sieve_enabled', [
-			'notnull' => false,
-			'default' => false
-		]);
-
-		$table = $schema->getTable('mail_classifiers');
-		$table->changeColumn('active', [
-			'notnull' => false,
-			'default' => false
-		]);
-
-		$table = $schema->getTable('mail_mailboxes');
-		$table->changeColumn('selectable', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('sync_in_background', [
-			'notnull' => false,
-			'default' => false
-		]);
-
-		$table = $schema->getTable('mail_messages');
-		$table->changeColumn('flag_answered', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_deleted', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_draft', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_flagged', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_seen', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_forwarded', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_junk', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_notjunk', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_important', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('flag_mdnsent', [
-			'notnull' => false,
-			'default' => false
-		]);
-		$table->changeColumn('structure_analyzed', [
-			'notnull' => false,
-			'default' => false
-		]);
+		if ($schema->hasTable('mail_accounts')) {
+			$table = $schema->getTable('mail_accounts');
+			$table->changeColumn('sieve_enabled', [
```

Comment:
but that column already existed in v1.9. Shouldn't this be moved to the other migration and instead here you change the mail_tags.is_default_tag?

Reply:
ah yes, sieve was released with 1.9, i forgot.
Do I even need the is_default_tag? Since this is in the 1.10 migration which wasn't released yet it should be fine to just change the create table statement.